### ISSUE TITLE
Choice of motor_speed coefficients

### ISSUE THREAD

**Issue by Jasper-Dong** [TheNoobInventor/lidarbot#7]
Hi, 
Thanks for your contribution in this project.
I have a question rather than an issue with your source code.

https://github.com/TheNoobInventor/lidarbot/blob/main/lidarbot_base/src/motor_encoder.c 
In this c file, you set the motor_speed coefficient as 1.65, so I would like to ask how did you calculate this value?
Thank you.

**Comment by TheNoobInventor** [comment]
Hello Jasper, 

I assume that you sorted out your previous issue.

Concerning the motor_speed coefficient value, I arrived at that by trial and error. Using a value of 1.0 made it hard for my robot to turn, so I gradually increased it and tested its navigation performance.

The motors I use have encoder ticks of approximately 1084 and using 1.65 seems to work well for me. There was someone that used a motor that had above 3000 encoder ticks and using the coefficient value of 1.0 was fine for him. So it depends on the motor you are using.

I hope this helps clarify things a bit.

**Comment by Jasper-Dong** [comment]
Hello,
Thanks for the reply.
Yes, the previous issue was addressed. I still don't know what is the root cause but I built the environment on another host machine with NVIDIA graphics instead of virtual machine and this issue doesn't show up anymore.
Back to the command coefficient, yes I had the same problem that it is really hard to turn left/right. But I was just wondering how did you calculate 1.65 as yours. But now it is clear.
Thanks again for your help.
